client/src/styles.css

```diff
@@ -132,7 +132,7 @@
 }
 
 /* DO scroll app category menu */
-.ant-layout-sider-children > header + div + ul {
+.ant-layout-sider-children > a + div + ul {
```

Comment:
Are you sure no `.ant-layout-sider-children > header + div + ul` use cases still exist?

Reply:
Fairly certain